src/Microsoft.AspNetCore.Razor.Evolution/DefaultRazorDirectiveClassifierPhase.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Razor.Evolution
+{
+    internal class DefaultRazorDirectiveClassifierPhase : RazorEnginePhaseBase, IRazorDirectiveClassifierPhase
+    {
+        public IRazorDirectiveClassifierPass[] Passes { get; private set; }
```

Comment:
I just left it there the way `DefaultRazorIRPhase` did it https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor.Evolution/DefaultRazorIRPhase.cs#L10